src/component/feature/CreateFeature/hooks/useFeatureForm.ts

```diff
@@ -42,7 +42,6 @@ const useFeatureForm = (
         return {
             type: type,
             name: name,
-            projectId: project,
```

Comment:
What do we actually use project for now?

Reply:
The form will probably still need to capture the projectId in order to call the correct API endpoint to create the feature toggle (projectId is part of the URL). We should just no send is as part of the json body anymore.

Reply:
We just removed the projectId from the body payload but we still keep it in the URL.